### ISSUE TITLE
How to access the result of explain query through JDBC ?

### ISSUE THREAD

**Issue by fuwhu** [ClickHouse/clickhouse-jdbc#530]
Problem description
We use JDBC to connect to ClickHouse, and we want to get the execution plan of specific SQL query using explain.
However, it seems we can not get the result of the explain query through JDBC.

Is there any mechanism that we can get the explain result or we need to add this feature somehow ?

**Comment by den-crane** [comment]
fix https://github.com/ClickHouse/clickhouse-jdbc/pull/529

**Comment by zhicwu** [comment]
Great work, thanks @den-crane! Just merged the PR. I'm thinking to roll out `0.2.5` release in the coming weeks focusing on bug fixes and small enhancement like this, and then start a new branch in parallel for refactoring.

**Comment by zhicwu** [comment]
Fixed on `develop` branch and will be merged into `master` after 0.2.5 release this weekend.
